recipe/meta.yaml

```
@@ -52,7 +52,7 @@ outputs:
         - {{ compiler('cxx') }}
       host:
         - python
-        - numpy 1.15
+        - numpy >=1.15
```

this should not be pinned this way.

it should be
```suggestion
        - numpy 1.15   # [not (aarch64 or ppc64le)]
        - numpy 1.16   # [aarch64 or ppc64le]
```

Otherwise it will build with numpy 1.17

---

and potentially add a comment stating that we never build numpy 1.15 for aarch64 or ppc64le

---

yup. For some reason I was seeing numpy as a runtime dep. Will change to your suggestion.